metadata_service/proxy/neo4j_proxy.py

```diff
@@ -965,3 +967,54 @@ def delete_table_relation_by_user(self, *,
             raise e
         finally:
             tx.close()
+
+    @timer_with_counter
+    def get_dashboard(self,
+                      id: str,
+                      ) -> DashboardDetailEntity:
+
+        get_dashboard_detail_query = textwrap.dedent(u"""
+        MATCH (d:Dashboard {key: $query_key})-[:DASHBOARD_OF]->(dg:Dashboardgroup)-[:DASHBOARD_GROUP_OF]->(c:Cluster)
+        OPTIONAL MATCH (d)-[:DESCRIPTION]->(description:Description)
+        OPTIONAL MATCH (d)-[:EXECUTED]->(last_exec:Execution) WHERE split(last_exec.key, '/')[5] = '_last_execution'
+        OPTIONAL MATCH (d)-[:LAST_UPDATED_AT]->(t:Timestamp)
+        OPTIONAL MATCH (d)-[:OWNER]->(owner:User)
+        OPTIONAL MATCH (d)-[:TAG]->(tag:Tag)
+        RETURN
+        c.name as cluster_name,
+        d.key as uri,
+        d.dashboard_url as url,
+        d.name as name,
+        toInteger(d.created_timestamp) as created_timestamp,
```

Comment:
could we store those timestamp as timestamp/int type during ingestion to avoid the cast here?

Reply:
> could we store those timestamp as timestamp/int type during ingestion to avoid the cast here?

Mmm.. I think there's no harm doing it here anyway as Neo4j does not guarantees the type of the field. WDYT? Also, casting here also make ingestion optimization easier later on.